### PR TITLE
capitalize ssn in "ssn is missing"

### PR DESCRIPTION
**Pull request by eidarus**
[ME-178907550](https://www.pivotaltracker.com/n/projects/2502930/stories/178907550)

### CODE REVIEW

**Comment by eidarus** [comment]
Screen Shot 2022-02-09 at 11 37 57 AM

**Review by bill-transue — CHANGES_REQUESTED** [review]
Use a transition or use `humanize` to get the formatting correct on this instead of doing customizing the view. This will make it so the correct format is used everywhere.

File: components/financial_assistance/app/views/financial_assistance/applicants/_dependent_form.html.erb [hunk 1]
```diff
@@ -14,6 +14,7 @@
               <ul>
                 <% @applicant.errors.full_messages.each do |msg| %>
                     <% msg = (msg == "Gender can't be blank" && EnrollRegistry.feature_enabled?(:gender_to_sex)) ? "Sex can't be blank" : msg %>
+                    <% msg = (msg == "ssn is missing") ? "SSN is missing" : msg %>
```
Comment (markgoho) [inline components/financial_assistance/app/views/financial_assistance/applicants/_dependent_form.html.erb:17]: are we missing some translations here as well?

**Review by markgoho — CHANGES_REQUESTED** [review]

File: components/financial_assistance/app/views/financial_assistance/applicants/_dependent_form.html.erb (hunk 1 above)
Comment (eidarus) [reply components/financial_assistance/app/views/financial_assistance/applicants/_dependent_form.html.erb]: The fix for gender can't be blank didn't end up being in the same file(s) as ssn is missing, so I'm gonna leave that change for another ticket/PR

**Review by bill-transue — APPROVED** [review]
This is the DRY Ruby way to do this. Good work @eidarus.

**Review by markgoho — APPROVED** [review]

**Pull request merged**